plugins/chat/assets/javascripts/discourse/connectors/home-logo-wrapper/chat-header-forum.gjs

```diff
@@ -0,0 +1,65 @@
+import Component from "@glimmer/component";
```

Comment:
I prefer to have 'stupid' connectors. Just have it render a component and the minimum necessary logic inside. I would even put the should render logic in the component, it's just easier to test and reuse.

Reply:
Yes, in the connector template:

```
<DiscourseChatHeader>
  {{yield}}
</DiscourseChatHeader>
```

Now in <DiscourseChatHeader /> you should be able to have the same content as you have currently in the connector, and when you call {{yield}} it should {{yield}} what was yielded in the connector.